apps/reg-scraper/package.json

```diff
@@ -44,4 +43,4 @@
     "tough-cookie": "^4.1.2",
     "winston": "^3.8.2"
   }
-}
+}
```

Comment:
not sure why EOF is missing every `package.json` lol

Reply:
My vscode config 💀, we should get some husky